monkvision/monkjs
# Feature/fix damage library

ALL CONTRIBUTIONS WILL BE REVIEWED BY AT LEAST ONE OF OUR FRONTEND TEAM MEMBERS.







## Technical description

The issue of the damages screen was due to using react-native-paper modules inside some components that are not wrapped in `PaperProvider` (react-native-paper provider).
The validate button in the menu is linked with the damages module from @monkvision/react-native-views (before it was just triggering the validate request, without any validation).

## Related to



This PR is related to [Apps are crashing when loading the DamageLibrary](https://monkvision.atlassian.net/browse/FE-249)

## Context and Motivation 

We shall make sure that all component using react-native-paper are already inside `PaperProvider`

## About Tests


Tested on the following devices

- iOS iPhone 8 device, iPhone 13 pro (emulator)
- Android pixel 2 xl
- Web browser chrome


Steps to reproduce

1. Go on production mode (you can toggle it on from metro).
2. Land on the damages screen.
3. Try add a damage.
4. Try validate the damage from the menu (a validation modal should show up).




## Screenshots (optional):

No screenshots.
*This Pull Request template has been written and generated by Monk JS repository.*
